### PR TITLE
Backupnaifbranch

### PR DESCRIPTION
**Pull request by Naif6003**
user signup

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/signup.jsp [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+						</div>
+					
+					
+						<div class="form-group label-static">
+							<label for="zip" class="control-label">Zipcode</label> <form:errors class="error" path="zip" />
+							<form:input path="zip" maxlength="12" cssClass="form-control" required="required" />
+						</div> 
+					
+					<button type="submit" class="btn btn-raised btn-primary">
+						<span class="glyphicon glyphicon-plus-sign"></span> ADD NEW USER
```
Comment (melaniemkwon) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: Update this button to 'Sign Up'

File: src/main/webapp/WEB-INF/jsp/signup.jsp [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+							<label for="username" class="control-label">Username</label>
+							<form:input path="username" maxlength="12" cssClass="form-control" required="required" />
+						</div>
+					
+						<div class="form-group label-static">
+							<label for="password" class="control-label">Password</label>
+							<form:password path="password" maxlength="12" cssClass="form-control" required="required" />
+						</div>
+						
+					<%-- <div class="form-group">
```
Comment (melaniemkwon) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: delete unneeded comments

File: src/main/webapp/WEB-INF/jsp/signup.jsp [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+<!-- Material Design for Bootstrap -->
+<script src="/formbuilder/assets/vendors/bootstrap-material-design/js/material.js"></script>
+<script src="/formbuilder/assets/vendors/bootstrap-material-design/js/ripples.min.js"></script>
+</head>
+<body>
+
+<div class="container">
+
+	<div class="col-md-offset-3 col-md-7">
+		<h3>Thank you for being part of our community</h3>
```
Comment (melaniemkwon) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: Change wording to 'Register New Account'

File: src/main/webapp/WEB-INF/jsp/signup.jsp [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+		
+		<div class="panel panel-primary">
+		
+			<div class="panel-heading">
+				<h4 class="panel-title">Signup</h4>
+			</div>
+				
+				<form:form modelAttribute="user" class="User" method="post" >
```
Comment (melaniemkwon) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: Formatting for form is off. Please add proper grid styling as documented in grid system for bootstrap.
https://getbootstrap.com/examples/grid/

File: src/main/webapp/WEB-INF/jsp/signup.jsp [hunk 5]
```diff
@@ -0,0 +1,117 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+
+
+<html>
+<head>
+<title>Sign up Page</title>
```
Comment (melaniemkwon) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: Remove title. This is already defined in tiles.xml

File: src/main/webapp/WEB-INF/jsp/signup.jsp [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+<link rel="stylesheet" href="/formbuilder/assets/vendors/bootstrap-3.3.7-dist/css/bootstrap.min.css">
```
Comment (melaniemkwon) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: Remove lines 10 to 21. All css and js are already imported in lpanel.jsp.

File: src/main/webapp/WEB-INF/jsp/signup.jsp (hunk 5 above)
Comment (Naif6003) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: done

File: src/main/webapp/WEB-INF/jsp/signup.jsp (hunk 6 above)
Comment (Naif6003) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: i added it for the assignment, the tiles were not working. Done

File: src/main/webapp/WEB-INF/jsp/signup.jsp (hunk 3 above)
Comment (Naif6003) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: done

File: src/main/webapp/WEB-INF/jsp/signup.jsp (hunk 4 above)
Comment (Naif6003) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: I will try using it but the formatting was working like that.

File: src/main/webapp/WEB-INF/jsp/signup.jsp (hunk 2 above)
Comment (Naif6003) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: done

File: src/main/webapp/WEB-INF/jsp/signup.jsp (hunk 1 above)
Comment (Naif6003) [inline src/main/webapp/WEB-INF/jsp/signup.jsp]: done

**Pull request merged**